Logical model examples without resourceType or id

jafeltra opened the pull request:
This PR adds a note to the Tips and Tricks section about logical model examples without resource type and id. I wasn't exactly sure where this should fit because it is really similar to the existing documentation. Let me know what you think.

Note: this approach is based on discussion on Zulip [here](https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/How.20do.20I.20get.20SUSHI.20to.20ignore.20a.20binary.20JSON.20logical.20instance.3F/near/407798801).

**Do not merge this PR until there is a SUSHI release that includes support for this.**

cmoesel reviewed (approved):
This looks good and is explained clearly. Thanks!

jafeltra commented:
I've updated this PR to use the current extension URL for IG resource format, since the one we had documented has been deprecated.

I've also updated it use a full canonical URL in the `resourceType` of our example since that is correct (SUSHI will still allow ids there, but we should document best practice). It had been bugging me for a while that our example Logical Model does not have an `id` element but the example Instance of it does (technically making the example invalid for that logical model), so I added an `id` element as well.

cmoesel commented on content/docs/SUSHI/tips/_index.md:
```diff
@@ -120,14 +120,15 @@ Logical: MyLM
 Title: "My LM"
 Description: "This is mine"
-* important 1..1 SU boolean "Is this resource important"
+* id 1..1 SU string "Identifier for the logical model"
```
Should this be the FHIR `id` type instead? It's a LM, so technically it can be _anything_. But since there is a FHIR primitive type called `id`, I think maybe we should use that to avoid any confusion.

jafeltra replied:
I totally agree. I had just forgotten that `id` was a primitive type, so I'll update that now.

jafeltra replied:
Added in [695c9f7](https://github.com/FSHSchool/site/pull/86/commits/695c9f7483e0b27ce8c0c24fe5fe71fd21e4d4c9).

cmoesel reviewed (commented):
I've asked a question / suggested a change. Technically the example is not _wrong_, so I couldn't make myself select `Request Changes` -- but it's probably a good idea to change it anyway.
